### PR TITLE
Migrate to Travis CI

### LINKED ISSUE #130: Migrate to Travis CI
Creating this bug mostly for myself and also /cc @MirkoBonadei because we had a discussion about this in #116. I know it's been a crazy long time but I'm getting to it now. Better late than never.

### PR DESCRIPTION
**Pull request by x1ddos**
This provides equivalent functionality:

- site build runs on every commit push and Pull Request
- the build is then copied to the prod website, if the commit push was on `gh-pages` branch

Additionally, html-proofer can be run for all builds.
It is currently disabled due to #131.

Closes #130.

### CODE REVIEW

File: _scripts/travis-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+set -e
+
+BUCKET="gs://weasel-webrtc"
+dir=$(dirname $0)
+
+[ "$TRAVIS_PULL_REQUEST" != "false" ] && exit 0
+[ "$TRAVIS_SECURE_ENV_VARS" != "true" ] && exit 0
+[ "$TRAVIS_BRANCH" != "gh-pages" ] && exit 0
+
+openssl aes-256-cbc -d \
```
Comment (MirkoBonadei) [inline _scripts/travis-deploy.sh]: I am wondering is this works on Travis. Is it possible to try this on this pull request even if the branch is not 'gh-pages'?

Comment (x1ddos) [reply _scripts/travis-deploy.sh]: I wrote this script specifically for travis. It's a new file. It does work. The working branch is travis-ci in this repo - that's how I made it work.

Comment (x1ddos) [reply _scripts/travis-deploy.sh]: I mean, look for previous runs on travis for travis-ci branch. You'll see that it works.

Comment (x1ddos) [reply _scripts/travis-deploy.sh]: btw, the command was generated by "travis encrypt-file". I didn't actually write it.

Comment (MirkoBonadei) [reply _scripts/travis-deploy.sh]: Oh, I see. I was missing that part about "travis encrypt-file" and I was trying to understand how does it work.

**Review by MirkoBonadei — APPROVED** [review]
lgtm.
Thanks for doing this.

**Comment by x1ddos** [comment]
Ah, it's a good point about travis encrypt-file. I should definitely add a comment in the script. Thanks.

Will do it in few hours, when I get to my laptop (on mobile now).

**Pull request merged**